tests/unit/test_redfish.py

```diff
@@ -485,7 +485,7 @@ def test__storage_root_uri(self):
     @patch(
         "prometheus_hardware_exporter.collectors.redfish.redfish_utilities.systems.get_system_ids"
     )
-    def test_get_storage_controller_data_success(
+    def test_get_storage_controller_data_success_with_storagecontrollers(
```

Comment:
nit: Maybe `test_get_storage_controller_data_success` and `test_get_storage_controller_data_success_with_non_standard_api` could be more descriptive test names?